src/library/librarycontrol.cpp

```diff
@@ -253,6 +253,19 @@ LibraryControl::LibraryControl(Library* pLibrary)
                     m_pSearchbox->slotMoveSelectedHistory(static_cast<int>(steps));
                 }
             });
+    m_pClearSearch = std::make_unique<ControlPushButton>(
+            ConfigKey("[Library]", "clear_search"));
+    connect(m_pClearSearch.get(),
+            &ControlPushButton::valueChanged,
+            this,
+            [this](double value) {
+                VERIFY_OR_DEBUG_ASSERT(m_pSearchbox) {
+                    return;
+                }
+                if (value >= 1.0) {
```

Comment:
```suggestion
                if (value > 0.0) {
```

Reply:
yes, I remember.
I changed it for all the searchbox controls.